src/Api/Vault/Controllers/SecurityTaskController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Bit.Api.Models.Response;
+using Bit.Api.Vault.Models.Response;
+using Bit.Core.Services;
+using Bit.Core.Vault.Enums;
+using Bit.Core.Vault.Queries;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Bit.Api.Vault.Controllers;
+
+[Route("tasks")]
+[Authorize("Application")]
+public class SecurityTaskController : Controller
```

Comment:
🎨 We should add a `[RequireFeature(FeatureFlagKeys.SecurityTasks)]` to the controller to hide it behind the feature flag.